### ISSUE TITLE
Adding applications to /opt/jboss/wildfly/domain/deployments/ doesn't work in domain-mode

### ISSUE THREAD

**Issue by rafabene** [jboss-dockerfiles/wildfly#12]
The instructions says " It can be /opt/jboss/wildfly/standalone/deployments/ or /opt/jboss/wildfly/domain/deployments/ depending on which mode you choose (standalone is default in the jboss/wildfly image -- see above)."

But adding applications to /opt/jboss/wildfly/domain/deployments/ doesn't work for Domain mode since there's no deployment scanner associated to domain mode

**Comment by goldmann** [comment]
Right, I must admit, I haven't tried to deploy in domain mode :) The easiest fix would be to correct the documentation. Care to send a PR?

**Comment by spolti** [comment]
PR https://github.com/jboss-dockerfiles/wildfly/pull/34
